smtp/main.go

```diff
@@ -145,7 +151,7 @@ func (s *smtpNotifier) sendSMTPNotification(build *cbpb.Build) error {
 	addr := fmt.Sprintf("%s:%s", s.mcfg.server, s.mcfg.port)
 	auth := smtp.PlainAuth("", s.mcfg.sender, s.mcfg.password, s.mcfg.server)
```

Comment:
Does this `PlainAuth` also need to have the `from` instead of `sender`?

Reply:
Since the sender is being used for authentication when initiating the session, PlainAuth only requires the sender.

Reply:
Sounds good, thanks for clarifying!